### PR TITLE
Updating event and operation states - issue #141

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by tsurdilo** [comment]
Changes based on issue https://github.com/cncf/wg-serverless/issues/141.

File: workflow/spec/schema/serverless-workflow-schema-01.json [hunk 1]
```diff
@@ -206,35 +206,36 @@
         "inputPath"
       ]
     },
-    "event": {
+    "onreceive": {
       "type": "object",
       "properties": {
         "condition": {
-          "description": "Boolean expression which consists of one or more Event operands and the Boolean operators",
+          "description": "Condition matching a received event with one or more defined triggers",
```
Comment (cathyhongzhang) [inline workflow/spec/schema/serverless-workflow-schema-01.json]: I think existing description gives more info about what condition means. The definition of condition is a Boolean expression of trigger events.

File: workflow/spec/schema/serverless-workflow-schema-01.json [hunk 2]
```diff
@@ -206,35 +206,36 @@
         "inputPath"
       ]
     },
-    "event": {
+    "onreceive": {
```
Comment (cathyhongzhang) [inline workflow/spec/schema/serverless-workflow-schema-01.json]: I feel onreveive is a more confusing name. onreceive what?

Comment (tsurdilo) [reply workflow/spec/schema/serverless-workflow-schema-01.json]: event(s) that match defined conditions

Comment (cathyhongzhang) [reply workflow/spec/schema/serverless-workflow-schema-01.json]: I understand, but "onreceive" seems to convey less info and more ambiguous. I don't feel this is a better change. Let's think more and try to come up with a better naming.

Comment (tsurdilo) [reply workflow/spec/schema/serverless-workflow-schema-01.json]: Any ideas? I like the current "onReceive". To me at least it's not confusing. 
@salaboy any suggestions?

**Comment by tsurdilo** [comment]
rebased against master

File: workflow/spec/schema/serverless-workflow-schema-01.json (hunk 1 above)
Comment (tsurdilo) [reply workflow/spec/schema/serverless-workflow-schema-01.json]: updated

**Comment by tsurdilo** [comment]
any ideas ..otherwise i'd like to move this forward please and listen to suggestions about different names when they come..we can update then.

**Comment by cathyhongzhang** [comment]
I think "eventAction" is a better name, which matches the description better "Define what events to act upon and actions to be performed". If you do not agree, my suggestion is to not change it (keep as is).

**Comment by tsurdilo** [comment]
@cathyhongzhang its an array of actions so "eventActions" ok for you ?

**Comment by cathyhongzhang** [comment]
Yes, eventActions is good for me.

**Comment by cathyhongzhang** [comment]
LGTM

**Pull request merged**